modules/gcp-vpc.adoc

```diff
@@ -0,0 +1,34 @@
+:_module-type: PROCEDURE
+//Specify the module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/gcp-private-connections.adoc
+
+[id="gcp-vpc_{context}"]
+= Configuring GCP VPC peering
+
+[role="_abstract"]
+A Virtual Private Cloud (VPC) peering connection is a networking connection between two VPCs that enables you to route traffic between them using private IPv4 addresses or IPv6 addresses. You can configure a {GCP} VPC containing an {product-title} cluster to peer with another GPC VPC network.
+
+[WARNING]
+====
+Private clusters cannot be fully deleted by the {cloud-redhat-com} if the VPC the cluster is installed in is peered.
```

Comment:
From the installer perspective, private clusters can be deleted by the **installer** if the VPC the cluster is installed in is peered.